CollAction/Models/AccountViewModels/RegisterViewModel.cs

```diff
@@ -29,5 +33,8 @@ public class RegisterViewModel
 
         [Display(Name = "Last name")]
         public string LastName { get; set; }
+
+        [Display(Name = "I would like to receive an update from CollAction every once in a while - don't worry, we like spam as little as you do! \U0001F642")]
```

Comment:
Doesn't really matter, but you don't need to escape the unicode, you can just paste 🙂 in the cs file.